corehq/util/global_request/middleware.py

```diff
@@ -7,4 +7,11 @@ def process_request(self, request):
         set_request(request)
 
     def process_response(self, request, response):
```

Comment:
May be one middleware should do only one function, should I make a new separate RememberDomainMiddleware instead?